echo10-construction/src/main.py

```diff
@@ -119,13 +119,24 @@ def get_granule_data(inputs, config):
             'THUMBNAIL_URL': browse_url,
             'PERPENDICULAR_BASELINE': granule_metadata['perpendicular_baseline'],
             'MISSION_NAME': mission,
+            'VERSION': granule_metadata['version'],
         },
         'input_granules': input_granules,
         'visible': 'true',
         'orderable': 'true',
         'online_access_url': online_access_url,
         'browse_url': browse_url,
     }
+
+    if granule_metadata['temporal_baseline_days'] != -1:
```

Comment:
we're not expecting sentinel values for these; they'll be optional in `sds_metadata`, so we just need to look for the existence of the key:
```suggestion
    if 'temporal_baseline_days' in granule_metadata:
```